coverage/lcov.info

```diff
@@ -1,168 +1,168 @@
 TN:
-SF:/home/doctor_mora_pol/dev/fox-staking-unified-history/contracts/RewardsDistributionRecipient.sol
+SF:/Users/pelsasser/shapeshift/fox-staking-3/contracts/RewardsDistributionRecipient.sol
 FN:8,onlyRewardsDistribution
 FNF:1
 FNH:1
-FNDA:289,onlyRewardsDistribution
-DA:9,289
-DA:10,289
+FNDA:283,onlyRewardsDistribution
+DA:9,283
+DA:13,283
 LF:2
 LH:2
-BRDA:9,1,0,289
+BRDA:9,1,0,283
 BRDA:9,1,1,0
 BRF:2
 BRH:1
 end_of_record
 TN:
-SF:/home/doctor_mora_pol/dev/fox-staking-unified-history/contracts/StakingRewards.sol
-FN:44,constructor
-FN:57,totalSupply
-FN:62,balanceOf
-FN:67,lastTimeRewardApplicable
-FN:72,rewardPerToken
-FN:83,earned
-FN:88,getRewardForDuration
-FN:95,stakeWithPermit
-FN:108,stake
-FN:117,withdraw
-FN:126,getReward
-FN:136,exit
-FN:144,notifyRewardAmount
-FN:167,updateReward
+SF:/Users/pelsasser/shapeshift/fox-staking-3/contracts/StakingRewards.sol
```

Comment:
lol look at us doxxing ourselves

Reply:
figures.  I should rename my user on my machine.  For now, maybe I will revert the coverage report changes, since the coverage actually hasn't changed from the original tests.

Reply:
all done.  will make for an easier PR for now as well, we can update coverage later if we choose to.